tests/settings/dev.py

```diff
@@ -1,4 +1,4 @@
-from .base import *
+from .base import *     # noqa: F401
```

Comment:
It might be better to do `# noqa: F401, F403` than to use the `ignore = F403` in tox.ini.